extensions/window/internal.m

```diff
@@ -390,6 +443,26 @@ static int window__close(lua_State* L) {
     return window_pressbutton(L, kAXCloseButtonAttribute);
 }
 
+/// hs.window:focusTab(index) -> bool
+/// Method
+/// Focuses the tab in the window's tab group at index, or the last tab if
+/// index is out of bounds. Returns true if a tab was pressed.
+///
+/// Parameters:
+///  * index - A number, true if the window should be set fullscreen, false if not
+///
+/// Returns:
+///  * The `hs.window` object
```

Comment:
wrong return description; per tag line description should probably read "true if the tab was successfully pressed, or false if there was a problem" or similar